go/api/base_client.go

```diff
@@ -7739,6 +7739,84 @@ func (client *baseClient) FCallReadOnlyWithKeysAndArgs(
 	return handleAnyResponse(result)
 }
 
+// InvokeScript executes a Lua script on the server.
+//
+// This function simplifies the process of invoking scripts on the server by using an object that
+// represents a Lua script. The script loading and execution will all be handled internally. If
+// the script has not already been loaded, it will be loaded automatically using the
+// `SCRIPT LOAD` command. After that, it will be invoked using the `EVALSHA`
+// command.
+//
+// See [LOAD] and [EVALSHA] for details.
+//
+// Parameters:
+//
+//	script - The Lua script to execute.
+//
+// Return value:
+//
+//	The result of the script execution.
+//
+// [LOAD]: https://valkey.io/commands/script-load/
+// [EVALSHA]: https://valkey.io/commands/evalsha/
+func (client *baseClient) InvokeScript(script options.Script) (any, error) {
+	response, err := client.executeScriptWithRoute(script.GetHash(), []string{}, []string{}, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	result, err := handleAnyResponse(response)
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+// InvokeScriptWithOptions executes a Lua script on the server with additional options.
```

Comment:
So
```suggestion
// Executes a Lua script on the server with additional options.
```
And the same for other docs.

Reply:
Updated.